c10/core/ScalarType.h

```diff
@@ -295,8 +295,8 @@ static inline bool isFloatingType(ScalarType t) {
 
 static inline bool isComplexType(ScalarType t) {
   return (
-      t == ScalarType::ComplexHalf || t == ScalarType::ComplexFloat ||
-      t == ScalarType::ComplexDouble);
+      t == ScalarType::ComplexFloat || t == ScalarType::ComplexDouble) ||
+      t == ScalarType::ComplexHalf;
```

Comment:
euh, nothing actually materially changed here right?

Reply:
nope but will just retain the original order to avoid any confusion

Reply:
@anjali411,

A note about ComplexHalf: I would love to see this supported, however it is a nightmare to do this on the CPU. I would suggest not even trying to support ComplexHalf with the kernel DISPATCH statements initially.